pulp/apis/core.py

```diff
@@ -33,10 +33,14 @@
 import os
 import sys
 
-try:
-    from time import process_time as clock
-except ImportError:
-    from time import clock
+
+if os.name == "posix":
+    from ..utilities import resource_clock as clock
```

Comment:
Under Linux, this is now measuring CPU time. 
If wall time is more appropriate to be silimar with windows, this case could be dropped entirely.